tests/std/tests/P0896R4_views_iota/test.cpp

```diff
@@ -0,0 +1,294 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <ranges>
+
+using namespace std;
+
+template <class W, class B>
+concept CanViewIota = requires(W w, B b) {
+    views::iota(w, b);
+};
+
+template <class R>
+concept CanSize = requires(R& r) {
+    ranges::size(r);
+};
+
+struct empty_type {};
+
+template <class T>
+constexpr void test_integral() {
+    constexpr T low        = 0;
+    constexpr T high       = 8;
+    constexpr T expected[] = {0, 1, 2, 3, 4, 5, 6, 7};
+
+    {
+        // Validate bounded (both upper and lower bounds exist) iota_view
+        using R = ranges::iota_view<T, T>;
+
+        // Validate type properties
+        static_assert(same_as<ranges::range_value_t<R>, T>);

```

Comment:
Include `<concepts>` for `same_as`?

Reply:
The standard requires `<ranges>` to include `<iterator>` which includes `<concepts>`. Weirdly, it also requires both `<ranges>` and `<iterator>` to include `<compare>`. 🤷